src/Plugin/Condition/AdoType.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Drupal\format_strawberryfield\Plugin\Condition;
+
+use Drupal\Core\Condition\ConditionPluginBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\webform\Entity\WebformOptions;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides the 'ADO Type' condition.
+ *
+ * @Condition(
+ *   id = "ado_type_condition",
+ *   label = @Translation("ADO Type"),
+ *   context_definitions = {
+ *     "node" = @ContextDefinition("entity:node", label = @Translation("node"))
+ *   }
+ * )
+ */
+class AdoType extends ConditionPluginBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Creates a new AdoType instance.
+   *
+   * @param array $configuration
+   *   The plugin configuration.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $type_options = [];
+    foreach(['schema_org_creative_works', 'schema_org_cw_collections'] as $webform_option_list_id) {
```

Comment:
@patdunlavey question.

What if we make this a Texture, like the "path" conditions work? Reason is, maybe users have more Webform options where to source the "types" from, or named differently. This of course could be also a setting provided by the Webform Strawberryfield module, since we also "sadly" do this hardcoded in AMI ... what do you think? We could also have a multi select where you choose first which webform_option (a select box) and then the type from there?

Reply:
I like the idea of making it a textarea field where the user in instructed to provide a comma-separated list of ADO types. Keep it simple AND fool-proof!

Reply:
@patdunlavey great. Maybe even a One Per line? That way we can reuse the code that splits/cleans that up from the other conditions that do that already and will also be a "I know how/common place" moment for users already understanding the multiple paths, each one in a single line, condition that comes standard with Drupal